kernel/kernel-defaults/src/test/scala/io/delta/kernel/defaults/DeltaTableReadsSuite.scala

```diff
@@ -150,16 +150,9 @@ class DeltaTableReadsSuite extends AnyFunSuite with TestUtils {
   }
 
   //////////////////////////////////////////////////////////////////////////////////
-  // Misc tests
+  // Table/Snapshot tests
   //////////////////////////////////////////////////////////////////////////////////
```

Comment:
This kind of header for categories of tests is good for `LogReplaySuite` (if there are categories).